modyn/supervisor/internal/grpc_handler.py

```diff
@@ -163,6 +163,7 @@ def get_time_at_storage(self) -> int:
         if not self.connected_to_storage:
             raise ConnectionError("Tried to fetch data from storage, but no connection was made.")
 
+        # pylint: disable=c-extension-no-member
```

Comment:
why this?

Reply:
that's a question to ask pylint, I don't know😅